### PR TITLE
Some more structure and consistency improvements

### PR DESCRIPTION
**Pull request by ErikSchierboom**

### CODE REVIEW

File: docs/porting-an-exercise.md [hunk 1]
```diff
@@ -66,25 +66,25 @@ master, which will make the WIP commit go away.
 You'll need to find the `slug` for the exercise. That's the URL-friendly identifier
 for an exercise that is used throughout all of Exercism.
 
-**The name of the exercise directory in the [x-common][] repository is the slug.**
+** The name of the exercise directory in the [x-common][] repository is the slug.**
```
Comment (kytrinyx) [inline docs/porting-an-exercise.md]: The space here will break the bolding.

**Comment by kytrinyx** [comment]
So good! :sparkles:

I've just got one request, and that is to fix the space issue in the bolded text.

**Comment by ErikSchierboom** [comment]
The PR has been updated!

**Comment by kytrinyx** [comment]
Thanks!

**Pull request merged**